src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
+
+  if (!response.ok) {
+    return [];
+  }
+
+  return response.json()
+    .catch(() => []);
```

Comment:
it would be nice to have try-catch block here. Have a look on this [page](https://timmousk.com/blog/typescript-try-catch/).